config/data.js

```diff
@@ -1209,6 +1226,17 @@ export default {
     'title': 'Population',
     'title_es': 'Población',
     'decimals': 0,
+    ,
+    'world_val': {
+      y_2010: 267583,
+      y_2015: 288817,
+      y_2016: 294618,
+      y_2017: 300865,
+      y_2018: 306457,
+      y_2019: 311848,
+      y_2020: 317185;
```

Comment:
```suggestion
      y_2020: 317185,
```

@libbymcclure this is failing tests b/c of the semicolon should be a comma